src/build/index.js

```diff
@@ -18,6 +18,8 @@ const buildMap = {
   [Kind.INPUT_OBJECT_TYPE_DEFINITION]: buildInputObject,
 };
 
+const SCHEMA_CONFIG_KEY = '__schema';
```

Comment:
You'll need to export this to be able to test the right configuration is being passed.